zuul.d/project-templates.yaml

```diff
@@ -60,6 +60,7 @@
         - podified-multinode-edpm-deployment-crc: *content_provider
         - cifmw-crc-podified-edpm-baremetal: *content_provider
         - podified-multinode-hci-deployment-crc: *content_provider
+        - podified-multinode-edpm-deployment-crc-tempest: *content_provider
```

Comment:
yeah its unrelated, but @rlandy wanted to add the podified-multinode-edpm-deployment-crc-tempest into github-check since we are not testing this job in check pipeline.

Reply:
ack thanks, but it looks like you removed it? (maybe you'll add it a followup?)